recipe/meta.yaml

```diff
@@ -9,13 +9,16 @@ source:
   url: https://cmake.org/files/v3.5/cmake-3.5.0-win32-x86.zip  # [win]
   fn: cmake-3.5.0-win32-x86.zip # [win]
 
+build:
+  number: 1
+
 test:
   commands:
     - cmake --version
 
 about:
   home: http://www.cmake.org/
-  license:  3-clause BSD
+  license: BSD 3-clause
```

Comment:
Puts the clause after like [OSI]( https://opensource.org/licenses/BSD-3-Clause ). Removes an extra space.